Foundation/Foundation.swift

```diff
@@ -0,0 +1,54 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+import CoreFoundation
+
+#if DEPLOYMENT_RUNTIME_SWIFT && canImport(Darwin)
```

Comment:
Isn't this due to the ObjC Foundation existing and not specific to Darwin?  Can we not do something like `@_runtime(_ObjC)` here? :-(

Reply:
I’ve tried really hard to confine private attributes elsewhere, but it is correct that the one you point is the most relevant condition.